onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1,21 +1,41 @@
-"""Test op correctness by comparing with PyTorch results."""
+"""Test op correctness by comparing with PyTorch results.
+
+## How to add a new operator test
+
+1. To enable test cases for an operator, add an entry to the
+`OPINFO_FUNCTION_MAPPING_SCRIPTED` map, run the test and follow instruction in
```

Comment:
what does "follow instruction" mean?

Reply:
The error message may say: "If the function is not trace_only, please move it to the 'OPINFO_FUNCTION_MAPPING_SCRIPTED' dict" for example